approve/index.php

```diff
@@ -13,36 +13,50 @@
 
 
 try {
-	$conn = new PDO("mysql:host=$servername;dbname=$dbname", $username, $password);
-	// set the PDO error mode to exception
-	$conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
-	//$sql = "SELECT purchaseID, item FROM Purchases WHERE Purchases.status = 'Requested'";
-	$sql = "SELECT DISTINCT p.purchaseID, p.item FROM Purchases p
-	INNER JOIN approval a on p.committee = a.committee
-	WHERE p.status = 'Requested'
-	AND a.username = '$usr'
-	AND (a.category = p.category OR a.category = '*')
-	AND p.cost <= (SELECT MAX(ap.ammount) FROM approval ap
-	WHERE ap.username = '$usr'
-	AND ap.committee = p.committee)";
-	//$stmt->execute();
-
-
-	foreach ($conn->query($sql) as $row) {
-		$items .= '<option value="';
-		$items .= $row['purchaseID'];
-		$items .= '">';
-		$items .= $row['item'];
-		$items .= '</option>\n';
-
-	}
-	//echo $items;
-
+    $conn = new PDO("mysql:host=$servername;dbname=$dbname", $username, $password);
+    // set the PDO error mode to exception
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    //$sql = "SELECT purchaseID, item FROM Purchases WHERE Purchases.status = 'Requested'";
+    $sql = "SELECT DISTINCT p.purchaseID, p.item FROM Purchases p
+    INNER JOIN approval a on p.committee = a.committee
+    WHERE p.status = 'Requested'
+    AND a.username = '$usr'
+    AND (a.category = p.category OR a.category = '*')
+    AND p.cost <= (SELECT MAX(ap.ammount) FROM approval ap
+    WHERE ap.username = '$usr'
+    AND ap.committee = p.committee)";
+    //$stmt->execute();
+
+
+    foreach ($conn->query($sql) as $row) {
+        $items .= '<option value="';
+        $items .= $row['purchaseID'];
+        $items .= '">';
+        $items .= $row['item'];
+        $items .= '</option>\n';
+    }
+    //echo $items;
+
+    $categorylist = "";
+    $committee = $_SESSION['committee'];
+    $sql = "SELECT category FROM `Budget` WHERE committee='$committee' AND year='2017-2018'";
```

Comment:
Did this for testing because that's what was there. I may end up adding the fiscal year change to this PR